src/assets/data/site-summary.json

```diff
@@ -1 +1 @@
-{"usa":{"lastDate":1648944000000,"totalCases":80056903,"totalDeaths":980027,"twoWeekPercentChangeInCases":-0.1098021084586966,"twoWeekPercentChangeInDeaths":-0.3784267805482849,"twoWeekPercentChangeInHospitalizations":-0.28214328114076165,"totalVaccinationsInitiated":255713962,"totalPopulation":331486822}}
+{"usa":{"lastDate":1649203200000,"totalCases":80174787,"totalDeaths":982264,"twoWeekPercentChangeInCases":-0.014078308520807504,"twoWeekPercentChangeInDeaths":-0.346996821147733,"twoWeekPercentChangeInHospitalizations":-0.2578300719748088,"totalVaccinationsInitiated":255873739,"totalPopulation":331486822}}
```

Comment:
I ran `yarn update-location-summaries` to update the summaries file with the community levels.  So that modified this file and summaries.json.